src/console-trace-link-exporter.ts

```diff
@@ -0,0 +1,122 @@
+import { ExportResult, ExportResultCode } from '@opentelemetry/core';
+import { ReadableSpan, SpanExporter } from '@opentelemetry/sdk-trace-base';
+import { HoneycombOptions } from './types';
+import { getTracesApiKey, isClassic } from './util';
+import {
+  FAILED_AUTH_FOR_LOCAL_VISUALIZATIONS,
+  MISSING_FIELDS_FOR_LOCAL_VISUALIZATIONS,
+} from './validate-options';
+import axios from 'axios';
+
+/**
+ * Builds and returns a {@link SpanExporter} that logs Honeycomb URLs for completed traces
+ *
+ * @remark This is not for production use.
+ * @param options The {@link HoneycombOptions} used to configure the exporter
+ * @returns the configured {@link ConsoleTraceLinkExporter} instance
+ */
+export function configureConsoleTraceLinkExporter(
+  options?: HoneycombOptions,
+): SpanExporter {
+  const apiKey = getTracesApiKey(options);
+  return new ConsoleTraceLinkExporter(options?.serviceName, apiKey);
+}
+
+/**
+ * A custom {@link SpanExporter} that logs Honeycomb URLs for completed traces.
+ *
+ * @remark This is not for production use.
+ */
+class ConsoleTraceLinkExporter implements SpanExporter {
+  private _traceUrl = '';
+
+  constructor(serviceName?: string, apikey?: string) {
+    if (!serviceName || !apikey) {
+      console.debug(MISSING_FIELDS_FOR_LOCAL_VISUALIZATIONS);
+      return;
+    }
+
+    const options = {
+      headers: {
+        'x-honeycomb-team': apikey,
+      },
+    };
+    axios.get('https://api.honeycomb.io/1/auth', options).then(
+      (resp) => {
+        if (resp.status === 200) {
+          const respData: AuthResponse = resp.data;
+          if (respData.team?.slug) {
+            this._traceUrl = buildTraceUrl(
+              apikey,
+              serviceName,
+              respData.team?.slug,
+              respData.environment?.slug,
+            );
+          } else {
+            console.log(FAILED_AUTH_FOR_LOCAL_VISUALIZATIONS);
+          }
+        }
+      },
+      () => {
+        console.log(FAILED_AUTH_FOR_LOCAL_VISUALIZATIONS);
+      },
+    );
+  }
+
+  export(
+    spans: ReadableSpan[],
+    resultCallback: (result: ExportResult) => void,
+  ): void {
+    if (this._traceUrl) {
+      spans.forEach((span) => {
+        // only log root spans (ones without a parent span)
+        if (!span.parentSpanId) {
+          console.log(
+            `Honeycomb link: ${this._traceUrl}=${span.spanContext().traceId}`,
```

Comment:
Didn't think the usual SDKLog prefix was necessary for this output since this was pretty self-evident

Reply:
I think it is still worth throwing it in there to keep things consistent!

Reply:
done!
![honeycomb link with prefix](https://github.com/honeycombio/honeycomb-opentelemetry-web/assets/29520003/c13c1006-d63a-40df-aa55-81bc4af43e54)